defaults/networking-defaults.yaml

```diff
@@ -5,19 +5,31 @@ defaults:
     power_model_constant_watt: 9.550000000
     power_model_variable_watt_per_mbps: 0.030000000
     streaming_resolution_per_device:
-      personal_computer: high
-      smartphone: medium
-      tablet: high
-      tv_system: ultra
+      digital-audio:
+        personal_computer: high
+        smartphone: medium
+        tablet: high
+        tv_system: high
+      streaming-video:
+        personal_computer: high
+        smartphone: medium
+        tablet: high
+        tv_system: ultra
   mobile:
     conventional_model_generic_kwh_per_gb: 0.140000000
     power_model_constant_watt: 1.200000000
     power_model_variable_watt_per_mbps: 1.530000000
     streaming_resolution_per_device:
-      personal_computer: high
-      smartphone: medium
-      tablet: high
-      tv_system: ultra
+      digital-audio:
+        personal_computer: high
+        smartphone: medium
+        tablet: high
+        tv_system: high
+      streaming-video:
+        personal_computer: high
+        smartphone: medium
+        tablet: high
+        tv_system: ultra
   unknown:
     conventional_model_generic_kwh_per_gb: 0.100000000
```

Comment:
I think we should add the ability to specify a blend when we don't know the network type. For instance TV, desktop, and tablet are probably 95% fixed network, but mobile phones are probably 75% mobile network (we should find a source for this). In other words the "unknown" networking type might not be necessary?

Reply:
👍 Going to be put into it's own PR